packages/builder/src/components/automation/SetupPanel/RowSelector.svelte

```diff
@@ -14,6 +14,11 @@
   export let block
   export let isTestModal
 
+  $: parsedBindings = bindings.map(binding => {
+    binding.icon = "ShareAndroid"
+    return binding
+  })
```

Comment:
It might be better here to return a new object inside the map function, just so we avoid mutating the real bindings array we pass around. Adding an icon like that is innocent enough but would still be better to avoid mutation.

e.g. just
```js
$: parsedBindings = bindings.map(binding => ({
  ...binding,
  icon: "ShareAndroid"
}))
```